source/switching_to_live/set_up_a_live_epdq_account/index.html.md.erb

```diff
@@ -10,61 +10,61 @@ Before you start, you must:
 - read about [what to do first](/switching_to_live/#before-you-start)
 - [request a live GOV.UK Pay account](/switching_to_live/#1-request-a-live-gov-uk-pay-account)
 
-You must also be an admin on your GOV.UK Pay account.
+You must also be an admin on your GOV.UK Pay account, or you will not see some of the following settings.
```

Comment:
I think I was unclear: I actually meant some settings are hidden if they’re not an admin on their ePDQ account (this does apply to their GOV.UK Pay account as well, but no more for ePDQ than any other PSP)